webpack/components/Table/TableWrapper.js

```diff
@@ -73,6 +75,8 @@ const TableWrapper = ({
     updatePagination(updatedPagination);
   };
 
+  const rowsCount = metadata?.subtotal ?? 0;
```

Comment:
Enjoying my first use of https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator :smile:

Reply:
ohhhh... me likey...